tests/unit/test_events.py

```diff
@@ -85,6 +87,11 @@ def pull_request(self):
         with open(DATA_DIR / "webhooks" / "github_pr_event.json", "r") as outfile:
             return json.load(outfile)
 
+    @pytest.fixture()
+    def merge_request(self):
+        with open(DATA_DIR / "webhooks" / "gitlab_mr_event.json", "r") as outfile:
+            return json.load(outfile)
```

Comment:
I think the `merge_request_webhook` fixture from `contest.py` should be visible here, do you really need to define another one here to get the same data?

Reply:
Also, can you please move the file from `webhooks/gitlab_mr_event.json` to `webhooks/gitlab/mr_event.json`?
I'm currently refactoring the fixtures and moving all the `webhooks/github_*.json` to `webhooks/github/*.json`.

Reply:
Okay will do it👍